src/components/EventObjectives.astro

```diff
@@ -0,0 +1,25 @@
+<div class="w-full">
+
+    <div class="grid md:grid-cols-3 md:grid-rows-2 grid-rows-4 mb-16">
+        <h1 class="font-bold col-span-full text-center md:text-6xl text-5xl my-auto">Objetivos do Evento</h1>
+        <div class="col-span-1 my-auto">
+            <img src="https://api.iconify.design/material-symbols:target.svg" class="md:w-1/3 w-1/5 invert-0 m-auto dark:invert" />
```

Comment:
What I was referring to on the other comments was this:
![image](https://github.com/Nucleo-Estudantes-Informatica-ISEP/codespring-2023/assets/103772884/13783827-ce47-44f1-8e8d-4799db05e599)

By clicking here:
![image](https://github.com/Nucleo-Estudantes-Informatica-ISEP/codespring-2023/assets/103772884/0a5d3203-dffd-443b-872d-7ffdc5c712ed)

You then just need to use the Icon component that comes from Astro. See how the instagram Icon from navbar is being set.

Let me know if you have any thoughts :smiley: